runtime/error_test.go

```diff
@@ -140,6 +140,44 @@ func TestRuntimeError(t *testing.T) {
 		)
 	})
 
+	t.Run("execution error with position", func(t *testing.T) {
+
+		t.Parallel()
+
+		runtime := newTestInterpreterRuntime()
+
+		script := []byte(`
+			pub fun main() {
+				let x: AnyStruct? = nil
+				let y = x!
+			}
+        `)
+
+		runtimeInterface := &testRuntimeInterface{}
+
+		location := common.ScriptLocation{0x1}
+
+		_, err := runtime.ExecuteScript(
+			Script{
+				Source: script,
+			},
+			Context{
+				Interface: runtimeInterface,
+				Location:  location,
+			},
+		)
+		require.EqualError(
```

Comment:
Was the output different before improving `interpreter.Error.Error()`?

Reply:
Yep, previously this just printed the `unexpectedly found nil while forcing an Optional value` message without including any of the location/position data